conf/traces-rel.json

```diff
@@ -0,0 +1,52 @@
+{
+  "ana": {
+    "int": {
+      "def_exc": true,
+      "interval": false,
+      "enums": true
+    },
+    "exp": {
+
+    }
+  },
+  "sem": {
+    "unknown_function": {
+      "invalidate": {
+        "globals": false
+      },
+      "spawn": true
+    },
+    "builtin_unreachable": {
+      "dead_code": true
+    }
+  },
+  "exp": {
+    "privatization": "mutex-meet",
+    "priv-distr-init": false,
+    "malloc": {
+      "wrappers": [
+        "kmalloc",
+        "__kmalloc",
+        "usb_alloc_urb",
+        "__builtin_alloca",
+        "kzalloc",
+
+        "ldv_malloc",
+
+        "kzalloc_node",
+        "ldv_zalloc",
+        "kmalloc_array",
+        "kcalloc",
+
+        "ldv_xmalloc",
+        "ldv_xzalloc",
+        "ldv_calloc"
```

Comment:
This list is now longer than the svcomp confs have. Where did these come from?

Reply:
They are also in the `traces.json` configuration, so I copied them from there

Reply:
Interesting, I'll look into whether they should also be part of the svcomp conf (for next year).